docs/api/quickstart/activitylog.md

```diff
@@ -0,0 +1,233 @@
+
+     ```HTTP
+    GET /spaces
+    ```
+
+    *Try in [Swagger](https://xyz.api.here.com/hub/static/swagger/#/Read_Spaces/getSpaces)*
+
+1. Post something into your newly created space:
+
+    ```HTTP
+    PUT /spaces/<newSpaceId>/features
+    ```
+
+    *Try in [Swagger](https://xyz.api.here.com/hub/static/swagger/#/Edit_Features/putFeatures)*
+
+    ```JSON
+    {
+      "type":"FeatureCollection",
+      "features":[
+        {
+          "id":"newFeatureId",
+          "type":"Feature",
+          "geometry":{
+            "type":"Point",
+            "coordinates":[1,0]
+          }
+        }
+      ]
+    }
+    ```
+
+    Check into your activity log space:
+
+    ```HTTP
+    GET /spaces/<activityLogSpaceId>/iterate
+    ```
+
+    *Try in [Swagger](https://xyz.api.here.com/hub/static/swagger/#/Read%20Features/iterateFeatures)*
+
+    This results in something like this
+
+    ```JSON
+    {
+      "type": "FeatureCollection",
+        ...
+      "features": [
+        {
+          "id": "<uuidOfFeature>",
+          "type": "Feature",
+          "properties": {
+            "@ns:com:here:xyz": {
+              "tags": [],
+              "space": "<activityLogSpaceId>",
+              "action": "SAVE",
+              "original": {
+                "id": "newFeatureId",
+                "space": "<newSpaceId>",
+                "createdAt": ...,
+                "invalidatedAt": ...,
+                "updatedAt": ...
+              },
+              "createdAt": ...,
+              "updatedAt": ...
+            }
+          },
+          "geometry": {
+            "type": "Point",
+            "coordinates": [
+              1,
+              0
+            ]
+          }
+        }
+      ]
+    }
+    ```
+
+## Search for specific feature
+
+You can search the Activity Log for a specific feature using its original id.
+This request returns an unsorted list of all revisions of the object.
+
+```HTTP
+GET /spaces/<activityLogSpaceId>/search?p.@ns:com:here:xyz.original.id="newFeatureId"
+```
+
+*Try in [Swagger](https://xyz.api.here.com/hub/static/swagger/#/Read%20Features/searchForFeatures)*
+
+Response
+
+```JSON
+{
+  "type": "FeatureCollection",
+  "features": [
+    {
+      "id": "<uuidOfFeature>",
+      "type": "Feature",
+      "properties": {
+        "@ns:com:here:xyz": {
+          "tags": [],
+          "space": "<activityLogSpaceId>",
+          "action": "SAVE",
+          "original": {
+            "id": "newFeatureId",
+            "space": "<newSpaceId>",
+            "createdAt": ...,
+            "invalidAt": ...,
+            "updatedAt": ...
+          },
+          "createdAt": ...,
+          "updatedAt": ...
+        }
+      },
+      "geometry": {
+        "type": "Point",
+        "coordinates": [
+          1,
+          0
+        ]
+      }
+    }
+  ]
+}
+```
+
+## Look up a specific revision of a feature
+
+You can search for a specific revision of a feature using the *uuid* of the feature:
+
```

Comment:
Yeah this confuses me. Does this return the most recent revision? Doesn't every revision have the same UUID from the original feature? What exactly is being searched here?